shared/macros-oval.jinja

```diff
@@ -7,17 +7,29 @@
 {{#
   High level macro which checks if a particular combination of parameter and value in a configuration file is set.
   This macro can take ten parameters:
-    - path (String): Path to the configuration file to be checked.
-    - prefix_regex (String): Regular expression to be used in the beginning of the OVAL text file content check.
-    - parameter (String): The parameter to be checked in the configuration file.
-    - value (String): The value to be checked. This can also be a regular expression (e.g: value1|value2 can match both values).
-    - separator_regex (String): Regular expression to be used as the separator of parameter and value in a configuration file. If spaces are allowed, this should be included in the regular expression.
-    - missing_parameter_pass (boolean): If set, the check will also pass if the parameter is not present in the configuration file (default is applied).
-    - application (String): The application which the configuration file is being checked. Can be any value and does not affect the actual OVAL check.
-    - multi_value (boolean): If set, it means that the parameter can accept multiple values and the expected value must be present in the current list of values.
-    - missing_config_file_fail (boolean): If set, the check will fail if the configuration is not existent in the system.
-    - section (String): If set, the parameter will be checked only within the given section defined by [section].
-    - quotes (String): If non-empty, one level of matching quotes is considered when checking the value. See comment of oval_line_in_file_state for more info.
+
```

Comment:
This is here to prove the concept that we can use somehow docstrings in jinja macros to document them.

This is the rendered result: https://ggbecker-content.readthedocs.io/en/readthedocs-support/jinja_macros/oval.html#template-oval_check_config_file

Reply:
Looks super nice